_templates/legacy.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * _templates/legacy.php
+ * Holds all legacy browser messages warnings
+ */
+
+$l10n->set_domain('layout');
+
+$user_agent = $_SERVER['HTTP_USER_AGENT'];
+
+$is_HTTP = (empty($_SERVER['HTTPS']) || $_SERVER['HTTPS'] !== 'on');
```

Comment:
This is only supposed to check for a non-empty string or "off". Some servers use "On", or simply `true` to mean it is on.